### PR TITLE
add python dependencies for SUIT

### PR DESCRIPTION
**Pull request by kaspar030**
This PR adds needed dependencies for SUIT (https://github.com/RIOT-OS/RIOT/pull/11818)

To test, make sure docker is correctly installed.

Then:

- check out this PR
- build the container (eg. ```docker build -t riotdocker_dev .```)
- check out #11818
- from the checkout root, run ```riot (suit-pr)]$ docker run -ti --rm -v $(pwd):/data/riotbuild -u $(id -u) riotdocker_dev /bin/bash```
- in the container, ```cd examples/suit_update; make suit/publish```

It should succeed.
Otherwise this shouldn't affect anything, as this is only an addition of python packages.

### CODE REVIEW

**Comment by kaspar030** [comment]
> Otherwise this shouldn't affect anything, as this is only an addition of python packages.

Basic functioning of the container is checked by Travis.

**Comment by fjmolinas** [comment]
Re-based on #75 it now succeeds. So lets wait for #75.

**Comment by emmanuelsearch** [comment]
@fjmolinas is this an ACK ;)
(now that #75 is merged)

**Comment by kaspar030** [comment]
> Doing as suggested fixed the issue.

This has been resolved.

File: requirements.txt [hunk 1]
```diff
@@ -4,3 +4,7 @@ flake8==3.5.0
 pexpect==4.2.1
 pycrypto==2.6.1
 pyserial==3.4
+ed25519==1.4
+cbor==1.0.0
+cryptography==2.6.1
+click==7.0
```
Comment (fjmolinas) [inline requirements.txt]: This can be removed.

Comment (emmanuelsearch) [reply requirements.txt]: @kaspar030 seems like an easy fix ;)
so let's merge this PR the last one preliminary to #11818 ?

Comment (kaspar030) [reply requirements.txt]: done

**Comment by fjmolinas** [comment]
GO!

**Pull request merged**